package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.18.0",
```

Comment:
you don't need `axios`
you could only use `fetch`, which is [native](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) on browsers